compiler/cpp/src/thrift/generate/t_rs_generator.cc

```diff
@@ -3212,8 +3221,12 @@ string t_rs_generator::service_call_handler_function_name(t_function* tfunc) {
   return "handle_" + rust_snake_case(tfunc->get_name());
 }
 
+string t_rs_generator::service_call_args_struct_name(t_function* tfunc) {
```

Comment:
Interesting that the definitions of these two functions are different. For example, why does `service_call_result_struct_name` include the `Result` suffix but the `service_call_args_struct_name` not?

I...assume (?) it's because the arglist has an `Args` suffix already? If so, could you please add a comment stating this?

Reply:
You're correct, the `Args` suffix is already present (coming directly from the thrift compiler structs). Good point, I'll add a comment to make it clear 🙂